babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -327,6 +327,34 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeJournalTable() {
+    final String sql = "create table foo, with journal table = baz (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, WITH JOURNAL TABLE = `BAZ` (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeJournalTableWithSource() {
```

Comment:
Maybe the name of this test could be?

testTableAttributeJournalTableWithCompoundIdentifier()